### DISCUSSION TITLE
Does inventory kamera support cloud platforms

### DISCUSSION THREAD

**Discussion by truong0235** [Andrewthe13th/Inventory_Kamera#417] [Q&A]
Like geforce now for example?

**Comment by ElmorenohWTF** [comment]
I don't know, I haven't tried it, but in the readme it doesn't mention anything about cloud platforms. I think it should work because in the end, Geforce Now supports virtual mouse and keyboard input. I haven't tested it, to get to know if it works, but if it doesn't, I think it would be a very good thing that could be added to the program, many people play games through cloud platforms.

**Comment by man90es** [comment]
There's no explicit support, but you can still try.

The main hurdle about working with cloud platforms is detecting the process. For the game that runs natively, it's always named either `GenshinImpact` or `YuanShen`. But for cloud platforms — who knows, it may run in the browser, or in a different app, so Kamera can't detect it without help. This is why in version 1.3.10, there was added an option to select the process manually.

You can find it in _Options_ -> _Update Executables_.
